ui/pages/confirmations/components/multilayer-fee-message/multi-layer-fee-message.test.js

```diff
@@ -5,11 +5,6 @@ import { renderWithProvider } from '../../../../../test/lib/render-helpers';
 import configureStore from '../../../../store/store';
 import MultilayerFeeMessage from './multi-layer-fee-message';
 
-jest.mock(
-  '../../../../helpers/utils/optimism/fetchEstimatedL1Fee',
-  () => '0x5',
```

Comment:
Do we need to replace this by adding the `layer1Fee` property to a mock transaction somewhere?

Reply:
Done.